rules/do-you-have-a-cookie-banner/rule.md

```diff
@@ -85,3 +85,4 @@ For example, the Tina.io cookie:
 Now that you have successfully created, customized and transmitted user cookies. Follow the following YouTube video to configure your Google Tag Manager.
 
 `youtube: yZjGzfWDc0Y`
+**Video: Guide to configuring consent code in Google Tag Manager**
```

Comment:
Hey @joshbermanssw 

1. Please add the time length as per https://www.ssw.com.au/rules/right-format-to-write-videos-time-length/

Reply:
✅